scripts/all-locales.js

```diff
@@ -28,6 +28,11 @@ var mkdirSync = function (path) {
   }
 }
 
+var copyFileSync = function(srcFile, destFile, encoding) {
+  var content = fs.readFileSync(srcFile, encoding);
+  fs.writeFileSync(destFile, content, encoding);
```

Comment:
I think this could probably be better written as `oldFile.pipe(newFile);`, although then you'ld be going down the async path…